data/avgi/avgi jobs.txt

```diff
@@ -1187,6 +1188,7 @@ mission "Avgi: Ergastirio Researchers 2"
 	name "Researchers to <planet>"
 	description "Take these <bunks> researchers to <destination> to work for a term on the laboratory station in exchange for <payment>."
 	source
+		distance 1 30
```

Comment:
Are you trying to say the source should be one to thirty jumps away from the destination?
This isn't how to do that.
You need:
```
near "whatever the destination system's name is" 1 30
```